### PR TITLE
Implements database to model.py class

### PR DESCRIPTION
**Pull request by whagestad**
@NateBigStone @xm6264jz

### CODE REVIEW

File: database.py [hunk 1]
```diff
@@ -5,30 +5,18 @@
...
-        conn.execute('CREATE TABLE IF NOT EXISTS records(restaurant TEXT, recipe TEXT, flickr TEXT, bookmarked INTEGER)')
+        conn.execute('CREATE TABLE IF NOT EXISTS records(restaurant TEXT, recipe TEXT, flickr TEXT)')
     conn.close()
 
 def add_data(yelp_response, edamam_response, flickr_response):
     bookmarked = 0
     try:
         with sqlite3.connect(db) as conn:
-            conn.execute('INSERT INTO records (restaurant, recipe, flickr, bookmarked) VALUES (?, ?, ?, ?)', (yelp_response, edamam_response, flickr_response, bookmarked))
+            conn.execute('INSERT INTO records (restaurant, recipe, flickr, bookmarked) VALUES (?, ?, ?)', (yelp_response, edamam_response, flickr_response))
```
Comment (xm6264jz) [inline database.py:15]: @whagestad You have bookmarked as variable and but you removed it from the values

**Review by xm6264jz — COMMENTED** [review]
@whagestad in the database.py files in some places you are using the bookmarked and in other places you are not

File: database.py (hunk 1 above)
Comment (whagestad) [reply database.py]: Ok, fixed that.

**Comment by whagestad** [comment]
This is ready to go, tested as a group in zoom meeting

**Review by NateBigStone — APPROVED** [review]

**Review by BrettKoenig04 — APPROVED** [review]

**Pull request merged**